docs/ref/contrib/postgres/indexes.txt

```diff
@@ -12,7 +12,7 @@ available from the ``django.contrib.postgres.indexes`` module.
 ``BrinIndex``
 =============
 
-.. class:: BrinIndex(fields=[], name=None, pages_per_range=None)
+.. class:: BrinIndex(**kwargs)
```

Comment:
I would emulate the model field reference docs which lists the field-specific options:
`FileField(upload_to=None, max_length=100, **options)`
so this would be
`BrinIndex(pages_per_range=None, **options)`